compactor/compactor.go

```diff
@@ -29,8 +29,7 @@ var (
 )
 
 const (
-	checkCompactionInterval   = 5 * time.Minute
-	executeCompactionInterval = time.Hour
+	checkCompactionInterval = 5 * time.Minute
```

Comment:
this const can also be removed.

Reply:
`checkCompactionInterval` is used `Revision` compactor. 
https://github.com/coreos/etcd/blob/master/compactor/revision.go#L64
I don't think it should be removed.

Reply:
ack